.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
```

Comment:
In my opinion, we should not use latest. It's so annoying when latest change and creates new behavior for seemingly no reason.

Reply:
Totally. I'll pin it to the same version Invenio Github actions are using, `ubuntu-20.04`, https://github.com/inveniosoftware/.github/blob/master/workflow-templates/pypi-publish.yml#L10